lib/mobility-core/src/Kernel/External/Notification/FCM/Flow.hs

```diff
@@ -164,10 +168,11 @@ notifyPerson ::
     MonadFlow m
   ) =>
   FCMConfig ->
+  m () ->
   FCMData a ->
   FCMNotificationRecipient ->
   m ()
-notifyPerson config msgData recipient = notifyPersonWithPriority config Nothing False msgData recipient EulerHS.Prelude.id
+notifyPerson config action msgData recipient = notifyPersonWithPriority config Nothing action False msgData recipient EulerHS.Prelude.id
```

Comment:
can you rename `action` at all places to something more meaningful, i.e. something like `unRegisteredFCMTokenHandler`.